packages/theme/src/theme/_forms.radio.scss

```diff
@@ -0,0 +1,74 @@
+$tt-radio-size: $form-checkbox-and-radio-size !default;
+$tt-radio-background-color: $wild-sand !default;
+$tt-radio-border-color: $dark-silver !default;
+$tt-radio-checked-background-color: $lochmara !default;
+$tt-radio-checked-border-color: shade($lochmara, 12) !default;
+$tt-radio-disabled-checked-background-color: $gray-light !default;
+$tt-radio-disabled-checked-border-color: $gray-light !default;
+$tt-radio-hover-border-color: $lochmara !default;
+$tt-radio-focus-border-color: $scooter !default;
+
+[type='radio'] {
+	&:before,
+	&:after,
+	+ *:before,
+	+ *:after {
+		@include transition(240ms);
+		width: $tt-radio-size;
+		height: $tt-radio-size;
+		border-radius: 50%;
+	}
+
+	&:before,
+	+ *:before {
+		@include scale(0);
+		background-color: $tt-radio-background-color;
+		z-index: 1;
+	}
+
+	&:after,
+	+ *:after {
+		background-color: $tt-radio-background-color;
+		border: 1px solid $tt-radio-border-color;
+	}
+
+	&:hover {
+		&:before,
+		+ *:before {
+			border-color: $tt-radio-hover-border-color;
+		}
+	}
+
+	&:focus {
+		outline: none;
+	}
+
+	&:checked:before,
+	&:checked + *:before {
+		@include scale(0.5);
+		background-color: $tt-radio-checked-background-color;
+	}
+
+	&:disabled:checked:before,
+	&:disabled:checked + *:before {
+		background-color: $tt-radio-disabled-checked-background-color;
+	}
+
+	&:focus + span:after,
+	&:checked:after,
+	&:checked + *:after {
+		border-color: $tt-radio-checked-border-color;
+	}
+
+	&:focus:after,
+	&:focus + span:after {
+		@include checkbox-outline;
```

Comment:
same variable here concerning the outline color which should be scooter in guideline (but doesn't look like it)